server/routes/checkout.js

```diff
@@ -0,0 +1,14 @@
+const router = require("express").Router();
+const stripe = require("stripe").Stripe(process.env.STRIPE_SECRET);
+
+router.get("/api/checkout", async (req, res) => {
+  const amount = 120;
```

Comment:
This is for one time payment correct?

Reply:
I would need to do more work to resolve things like checking whether user has paid before